SamplePlugin/SamplePlugin.cs

```diff
@@ -19,7 +19,7 @@ public class SamplePlugin : PluginBase
         // Dependencies will be injected automatically, if dependency class and field/property names match
         // Plugins won't load until all their required dependencies are added
         // Optional dependencies may be injected at any time, if at all
-        [Dependency(MinVersion = "2.0", Optional = true), Alias("Sample Remote Plugin")]
```

Comment:
Same here for previous comment. These were explicit for clarity.

Reply:
Same with this